lib/fxpay/settings.js

```diff
@@ -81,12 +83,19 @@
   };
 
   exports.configure = function settings_configure(newSettings, opt) {
+    //
+    // Configure new settings values.
+    //
     opt = opt || {};
+
+    // Reset existing configuration.
     if (opt.reset) {
       for (var def in defaultSettings) {
         exports[def] = defaultSettings[def];
       }
     }
+
+    // Merge new values into existing configuration.
     for (var param in newSettings) {
       if (typeof exports[param] === 'undefined') {
         exports.log.error('configure() received an unknown setting:', param);
```

Comment:
The line after this is also a problem as it's using `exports.onerror` before that's been defined.